### DISCUSSION TITLE
Playlist bug?

### DISCUSSION THREAD

**Discussion by Guldstrand** [AzuraCast/AzuraCast#5883] [Q&A]
A scheduled playlist takes over the default playlist (and only repeats the songs in that list), although the scheduled list has a very low "weight".
Is this a bug?

**Comment by BusterNeece** [comment]
It's intentional; scheduled playlists are supposed to take priority over "general rotation" playlists, which are intended to play whenever a scheduled playlist isn't present or valid.

**Comment by Guldstrand** [comment]
Ah, i see.
What's the best for me to do then?
When i want to play a christmas-list sertain period/dates, together with my
standard (non scheduled) playlist?


Regards..
Daniel

(skickat från min mobiltelefon)

Den fre 25 nov. 2022 04:36Buster Neece ***@***.***> skrev:
